docs/platform/quickstart.mdx

```diff
@@ -130,10 +130,10 @@ messages = [
 ]
 
 # The default output_format is v1.0
-client.add(messages, user_id="alex123", session_id="trip-planning-2024", output_format="v1.0")
+client.add(messages, user_id="alex123", run_id="trip-planning-2024", output_format="v1.0")
```

Comment:
Can you please add a note in this section in docs that `session_id` is deprecated and use `run_id`.